nextjs-starter/components/tags.jsx

```diff
@@ -0,0 +1,24 @@
+export const Tags = ({ tags }) => {
+  return (
+    <p className="mt-2 text-sm leading-[1.25rem] text-theme-bg-black">
+      {/* {tags.length ? (
```

Comment:
Could we remove this component completely? Otherwise a comment giving context would be great

Reply:
Accidentally kept this commented out when I was debugging something. I've uncommented it. Good catch